src/tests/multihost/sssd/testlib/common/custom_log.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+import os
+from collections import defaultdict
+
+import pytest
+
+
+@pytest.hookimpl(tryfirst=True)
+def pytest_configure(config: pytest.Config) -> None:
+    p = CustomLogPlugin(config)
+    config.pluginmanager.register(p, 'custom_log_plugin')
+
+
+class CustomLogPlugin:
+    """
+    Custom log class to remove log from console
```

Comment:
I'd update the docstring here to reflect that you can also duplicate between console and pre-test log files.

Reply:
Done.